### ISSUE TITLE
[QUESTION] Upgrade to vault 1.0

### ISSUE THREAD

**Issue by azman0101** [kelseyhightower/vault-on-google-kubernetes-engine#13]
Do you plan to upgrade or accept PR to upgrade this repo to vault 1.0 ?

https://www.hashicorp.com/blog/vault-1-0

**Comment by sethvargo** [comment]
Yes - we were waiting for the final release (which just happened yesterday)
